R/create_country_codes.R

```diff
@@ -0,0 +1,307 @@
+#' Country Code Lookup
+#'
+#' @description
+#' These pre-defined country codes are sourced from
+#' [ISO 3166 Standards](https://www.iso.org/iso-3166-country-codes.html).
+#' See also [Wikipedia](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3).
+#'
+#' @details
+#'
+#' `country_code` is the 3-letter ISO 3166-1 county code commonly found in the
+#' ADSL `COUNTRY` variable.
+#' `country_name` is the country long name corresponding to to the 3-letter code.
+#' `country_number` is the numeric code corresponding to an alphabetic sorting of
+#' the 3-letter codes.
+#'
+#' To see the entire table in the console, run `print(country_code_lookup)`.
+#'
+#' @seealso [dose_freq_lookup]
+#'
+#' @export
+#'
+#' @keywords metadata
+#'
+#' @family metadata
+#'
+#' @examples
+#' library(tibble)
+#' library(dplyr, warn.conflicts = FALSE)
+#' library(lubridate)
+#'
+#' # Create reference dataset for periods
+#' adsl <- tribble( #' adsl <- tribble(
```

Comment:
I think there is a wayward `#'` in here?

Reply:
@jeffreyad  could you check on this example?  Is the `#'` needed?  after that we can merge